packages/unchained-client/src/utxo/index.ts

```diff
@@ -0,0 +1,5 @@
+export type { utxo as types } from '@shapeshiftoss/common-api'
+export { BaseTransactionParser } from './parser'
+
+export * as bitcoin from './bitcoin'
+export * as dogecoin from './dogecoin'
```

Comment:
litecoin?

Reply:
addressed in https://github.com/shapeshift/lib/pull/914/commits/080c21f5f8f5aa8c96b55e31126c9b828da30d52